src/sha256t.rs

```diff
@@ -22,7 +22,7 @@ use Hash as HashTrait;
 use Error;
 
 /// Trait representing a tag that can be used as a context for SHA256t hashes.
-pub trait Tag: Copy + Ord + Default + ::core::hash::Hash {
+pub trait Tag: Copy + Ord + Default + HashTrait {
```

Comment:
This is not correct. `Tag` is not a hash...

Reply:
I had a compilation issues before I changed this, will re-check what it was. In my project I'm using tagged hashes the same way I use other hashes, and some macro, generic by a hash type, was failing due to this line (of Tag being not Hash, but being a part of `sha256t::Hash`). Will double-check that and follow up.

Reply:
Ok, I messed up; the problem was caused by another factor. Closing the request